apps/krew-website/src/components/Footer.astro

```diff
@@ -0,0 +1,23 @@
+---
+
+const mainScript = Astro.resolve('../scripts/script.js');
+
+---
+<footer class='bg-red-600 p-4 fixed bottom-0 left-0 right-0 z-10'>
+  <div class='max-w-7xl mx-auto px-4 sm:px-6'>
+    <div class='text-white'>
+      <div class="text-white flex flex-row items-center">
+        <div class='pr-2'>©</div>
+        <div class='pr-2'><img class='d-inline' src='/assets/move4mobile-white.svg' alt='Move4Mobile'></div>
+        <div class='pr-2'>2021</div>
+        <div class='ml-auto'><a href='#' id='back2Top'>
+          <svg xmlns="http://www.w3.org/2000/svg" class="h-6 w-6" fill="none" viewBox="0 0 24 24" stroke="currentColor">
+            <path stroke-linecap="round" stroke-linejoin="round" stroke-width="2" d="M9 11l3-3m0 0l3 3m-3-3v8m0-13a9 9 0 110 18 9 9 0 010-18z" />
+          </svg>
+        </a></div>
+      </div>
+    </div>
+  </div>
+</footer>
```

Comment:
Currently there's quite some inconsistency regarding quotes. Doesn't the IDE/Prettier already handle this on save?
Anyway, my preference would be all HTML attribute-quotes are `""` (double quotes).

Reply:
My preference is double as well, Webstorm converts it to single... I'll have a look at VC, there is an astro plugin available.